### PR TITLE
Fix collection delete

### PR DESCRIPTION
**Pull request by jbwexler**

### CODE REVIEW

**Comment by chrisfilo** [comment]
could you add a test?

**Comment by jbwexler** [comment]
i've added a test. I also added a test_media_root directory and a function to clear it that can be used in a the TearDown function of a test.

**Comment by chrisfilo** [comment]
Could you rename methods.py to utils.py? It will follow the convention we use elsewhere (and those are technically functions not methods). Sorry for nitpicking!

**Comment by jbwexler** [comment]
done

File: neurovault/apps/statmaps/tests/test_nidm.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 from django.core.files.uploadedfile import SimpleUploadedFile
 from neurovault.apps.statmaps.nidm_results import NIDMUpload
 from neurovault.apps.statmaps.forms import NIDMResultsForm
+from .methods import clearTestMediaRoot
```
Comment (chrisfilo) [inline neurovault/apps/statmaps/tests/test_nidm.py]: shouldn't this be .utils now?

File: neurovault/apps/statmaps/tests/test_feat.py [hunk 2]
```diff
@@ -9,6 +9,7 @@
 from nidmfsl.fsl_exporter.fsl_exporter import FSLtoNIDMExporter
 import urllib
 import zipfile
+from .methods import clearTestMediaRoot
```
Comment (chrisfilo) [inline neurovault/apps/statmaps/tests/test_feat.py]: shouldn't this be .utils now?

File: neurovault/apps/statmaps/tests/test_comparison.py [hunk 3]
```diff
@@ -10,6 +10,8 @@
 import shutil
 from neurovault.apps.statmaps.utils import split_afni4D_to_3D
 import nibabel
+from .methods import clearTestMediaRoot
```
Comment (chrisfilo) [inline neurovault/apps/statmaps/tests/test_comparison.py]: shouldn't this be .utils now?

File: neurovault/apps/statmaps/tests/test_collections.py [hunk 4]
```diff
@@ -1,12 +1,18 @@
...
-from neurovault.apps.statmaps.models import Collection,User
+from neurovault.apps.statmaps.models import Collection,User, Atlas
+from django.core.files.uploadedfile import SimpleUploadedFile
 from uuid import uuid4
 import tempfile
 import os
 import shutil
 from neurovault.apps.statmaps.utils import detect_afni4D, split_afni4D_to_3D
 import nibabel
+from .methods import clearTestMediaRoot
```
Comment (chrisfilo) [inline neurovault/apps/statmaps/tests/test_collections.py]: shouldn't this be .utils now?

File: neurovault/apps/statmaps/tests/test_api.py [hunk 5]
```diff
@@ -7,6 +7,7 @@
 from django.core.files.uploadedfile import SimpleUploadedFile
 from neurovault.apps.statmaps.forms import NIDMResultsForm
 import json
+from .methods import clearTestMediaRoot
```
Comment (chrisfilo) [inline neurovault/apps/statmaps/tests/test_api.py]: shouldn't this be .utils now?

**Comment by jbwexler** [comment]
whoops sorry about that. just changed the imports and all the tests pass now

**Pull request merged**